components/canvas/Canvas.tsx

```diff
@@ -24,3 +20,26 @@ function Canvas() {
   );
 }
 export default Canvas;
+
+const albert = { x: 1360, y: 250, width: 50, height: 10 };
+
+function moveAlbert(context) {
```

Comment:
Ich würde die Funktion anders benennen, da sie mehr macht als Albert nur zu bewegen. Du malst ihn ja auch. Mein Vorschlag wäre etwas in Richtung create and move Altert, oder draw and move Albert.